plugins/osdn/ovs/controller.go

```diff
@@ -346,12 +345,11 @@ func (c *FlowController) AddOFRules(nodeIP, nodeSubnetCIDR, localIP string) erro
 	}
 
 	glog.V(5).Infof("AddOFRules for %s", nodeIP)
-	cookie := generateCookie(nodeIP)
 	otx := ovs.NewTransaction(BR)
 
-	otx.AddFlow("table=1, cookie=0x%s, priority=100, tun_src=%s, actions=goto_table:5", cookie, nodeIP)
-	otx.AddFlow("table=8, cookie=0x%s, priority=100, arp, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, nodeSubnetCIDR, nodeIP)
-	otx.AddFlow("table=8, cookie=0x%s, priority=100, ip, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", cookie, nodeSubnetCIDR, nodeIP)
+	otx.AddFlow("table=1, cookie=0x0, priority=100, tun_src=%s, actions=goto_table:5", nodeIP)
+	otx.AddFlow("table=8, cookie=0x0, priority=100, arp, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", nodeSubnetCIDR, nodeIP)
+	otx.AddFlow("table=8, cookie=0x0, priority=100, ip, nw_dst=%s, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31],set_field:%s->tun_dst,output:1", nodeSubnetCIDR, nodeIP)
```

Comment:
If we are not setting cookie, we don't have to specify it right?
otx.AddFlow("table=8, priority=100, ...)